src/adafruit_blinka_raspberry_pi5_piomatter/click.py

```diff
@@ -73,9 +81,13 @@ def wrapper(f: click.decorators.FC):
                 help="The overall orientation (rotation) of the panels"
             )(f)
         if n_planes is not None:
-            f = click.option("--num-planes", "n_planes", default=n_planes, help="The number of bit planes (color depth. Lower values can improve refresh rate in frames per second")(f)
+            f = click.option("--num-planes", "n_planes", default=n_planes, help="The number of bit planes (color depth). Lower values can improve refresh rate in frames per second")(f)
+        if n_temporal_planes is not None:
```

Comment:
@jepler Does this argument (and/or the others near here) need to be conditional with this `not None` check? 

If I am understanding correctly this means that the `--num-temporal-planes` argument is not valid unless the example code script includes something like:
```
@piomatter_click.standard_options(n_lanes=2, n_temporal_planes=4)
```
Is that understanding correct? 

In the case where the code provides a value in the click.standard_option declaration, and the user also passes `--num-temporal-planes` which would take precedent?

If it's possible have them be just CLI arguments and not need to be declared in the example scripts also I think that would make their usage less complex.

Reply:
Yes. This new argument is only added to click if it's specified as non-None.

My thinking, which may be making more trouble than help: If the new command line argument is added to click by default, then any old code that doesn't have the python function parameter of the same name will suddenly break.

We're in alpha releases in this repo, so it's probably OK to just break things for now. The amount of code outside the repo that is using `protomatter_click.standard_options` is essentially zero.

would you rather I change it so it's included by default?

Reply:
Ah, I see. I do think it would be good to have it included by default. Agree there is likely little else using it at the moment so it's okay to have it break.